packages/http-server/src/__tests__/server.oas.spec.ts

```diff
@@ -29,33 +29,39 @@ async function instantiatePrism(specPath: string) {
     errors: false,
     cors: true,
   });
+
+  const address = await server.listen(port, '127.0.0.1');
+
+  return {
+    close: server.close.bind(server),
+    address,
+  };
 }
 
 describe('GET /pet?__server', () => {
-  let server: IPrismHttpServer;
+  let server: { close: () => Promise<void>; address: string };
 
-  beforeAll(async () => {
-    server = await instantiatePrism(resolve(__dirname, 'fixtures', 'templated-server-example.oas3.yaml'));
+  beforeEach(async () => {
+    server = await instantiatePrism(30002, resolve(__dirname, 'fixtures', 'templated-server-example.oas3.yaml'));
```

Comment:
I vote to always use the same port — the reason is that this will help us discover cases of server handles not released/freed up quickly instead of blowing up CircleCI or our computers

Reply:
Test suites can't be on the same port because they run in parallel and will occasionally fail.

`    listen EADDRINUSE: address already in use 127.0.0.1:30000`

Reply:
Updated so the port inside test suite is the same

Reply:
I also commented on those ports. The Next Generation which will work on Prism might fall into hard to debug problem